app/lib/gpDetailsParser.js

```diff
@@ -1,6 +1,8 @@
 const xml2js = require('xml2js');
 const jsonQuery = require('json-query');
 const stripPrefix = require('xml2js/lib/processors').stripPrefix;
+const assert = require('assert');
+const Verror = require('verror');
```

Comment:
This is pretty nitpicky but it looks like the casing should be `VError` from the npm docs https://www.npmjs.com/package/verror